### ISSUE TITLE
Deploying to Truffle

### ISSUE THREAD

**Issue by anthonyemberley** [nigeon/CryptoZombies#1]
Did you have any issues deploying to truffle? When I do:

`truffle compile
truffle migrate`

I keep getting this error:

`
Deploying ZombieOwnership...
Error encountered, bailing. Network state unknown. Review successful transactions manually.
Error: The contract code couldn't be stored, please check your gas amount.
`

I've tried upping the gas on the ZombieOwnership in the migrations tab but can't get it working

**Comment by nigeon** [comment]
Nope, but looks like [other people are experiencing similar errors](https://www.google.es/search?q=Error+encountered%2C+bailing.+Network+state&oq=Error+encountered%2C+bailing.+Network+state&aqs=chrome..69i57j35i39j69i61l3.356j0j8)

I've no problem in MacOs Sierra (10.12.6) with:
Truffle v4.0.4
Ganache v.1.1.0
